src/flows/components/Sidebar.tsx

```diff
@@ -285,6 +343,7 @@ const Sidebar: FC = () => {
       className="flow-sidebar--dropdownmenu"
       style={{width: '200px'}}
     >
+      {showDownload && <DownloadAsPDF pipeID={pipeID} />}
```

Comment:
feels like you can just lazyload the function and call it instead of this "load a component and useEffect on mount" thing